typescript/ts-node-introduction/src/org/jetbrains/dukat/nodeIntroduction/introduceNodes.kt

```diff
@@ -403,15 +403,7 @@ private class LowerDeclarationsToNodes(private val fileName: String, private val
             is CallSignatureDeclaration -> listOf(
                     FunctionNode(
                             QualifierEntity(name, IdentifierEntity("invoke")),
-                            convertParameters(declaration.parameters.map { param ->
-                                val initializer = if (param.initializer?.kind?.isSimpleType("definedExternally") == true) {
-                                    ExpressionDeclaration(TypeDeclaration(IdentifierEntity("null"), emptyList()), null)
-                                } else {
-                                    param.initializer
-                                }
-
-                                param.copy(initializer = initializer)
-                            }),
+                            convertParameters(declaration.parameters.map { it.copy(initializer = null) }),
```

Comment:
The original logic here seems questionable to me. I can't quite make out why a default value in an invoke function can't also just be "definedExternally", and is replaced here with "null". It seems like modification to the initializers here is not actually necessary.